src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/CSharpSyntaxContext.cs

```diff
@@ -97,6 +97,7 @@ internal sealed class CSharpSyntaxContext : SyntaxContext
             bool isPreProcessorExpressionContext,
             bool isPreProcessorKeywordContext,
             bool isPrimaryFunctionExpressionContext,
+            bool isRightAfterUsingOrImport,
```

Comment:
```suggestion
            bool isRightAfterUsingOrImportDirective,
```

RIght now it sounds like you're after the keyword.